cypress/support/commands.js

```diff
@@ -1,3 +1,12 @@
+Cypress.Commands.add('findByType', (type) => {
+    cy.get(`[type="${type}"]`);
+ });
+ 
```

Comment:
There are two empty lines between the commands. It is recommended to have just one empty line between code blocks for consistency and readability 😊